site/src/pages/WorkspacePage/WorkspacePage.tsx

```diff
@@ -102,27 +103,26 @@ export const WorkspacePage: FC = () => {
     workspaceQuery.error ?? templateQuery.error ?? permissionsQuery.error;
   const isLoading = !workspace || !template || !permissions;
 
-  if (pageError) {
-    return (
-      <Margins>
-        <ErrorAlert
-          error={pageError}
-          css={{ marginTop: 16, marginBottom: 16 }}
-        />
-      </Margins>
-    );
-  }
-
-  if (isLoading) {
-    return <Loader />;
-  }
-
   return (
-    <WorkspaceReadyPage
-      workspace={workspace}
-      template={template}
-      permissions={permissions}
-    />
+    <div css={{ height: "100%", display: "flex", flexDirection: "column" }}>
```

Comment:
I very much preferred the more explicit early-return style that this component used before.

Reply:
Me too, but since now it needs to wrap other elements I didn't find a better way but this is temporary until I remove the navbar element in the next refactoring.